### PR TITLE
Move Entry to solved Section

### LINKED ISSUE #504: Move FAQ entry regarding backup problems on iOS to solved Section
## Where to find the issue

https://www.coronawarn.app/en/faq/?search=Backup#migration_Apple
https://www.coronawarn.app/de/faq/?search=Backup#migration_Apple

## Describe the issue & Suggested Change 

Since the App now works correctly after restoring it from a Backup, this entry should be moved to the Solved Section.

### PR DESCRIPTION
**Pull request by Ein-Tim**
Closes #504 
@svengabr could you take a look at this PR, hope this time there are no syntax errors in it 👍
Thank you for all your great work!

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -765,6 +755,16 @@
                 "id": "resolved",
                 "title": "Geklärt",
                 "accordion": [
+                    {
+                        "title": "[Apple/iOS]: Migration auf neues Gerät bzw. Backup/Restore - App funktioniert nicht",
+                        "anchor": "migration_Apple",
+                        "active": true,
+                        "textblock": [
+                            "AKTUELL: Dieses Problem wurde mit Version 1.5.0 behoben. Wenn Sie Ihre Daten über iTunes auf ein neues Gerät übertragen oder ein Backup wiederhergestellt haben, funktioniert die App nicht richtig. Auf den Startseiten landen Sie in einer Endlosschleife und/oder die Risiko-Ermittlung lässt sich nicht einschalten. Der Risikostatus wird nicht angezeigt.",
```
Comment (heinezen) [inline src/data/faq_de.json:763]: Just for consistency with other entries: Can you format AKTUELL like this:

```
<b>AKTUELL</b><br/>
```

and add a separator at the end: `<hr/>`?

File: src/data/faq.json [hunk 2]
```diff
@@ -764,6 +754,16 @@
                 "title": "Resolved",
                 "id": "resolved",
                 "accordion": [
+			 {
+                        "title": "[Apple/iOS]: Migrating to new phone and backup and restore - app does not work",
+                        "anchor": "migration_Apple",
+                        "active": true,
+                        "textblock": [
+                            "UPDATE: This problem has been solved with version 1.5.0. If you transfer your data to a new iPhone using iTunes, or restore a backup, the app does not work properly. On the start screens, you end up in a loop, or the exposure logging cannot be activated. The risk status is not displayed.",
```
Comment (heinezen) [inline src/data/faq.json:762]: see the other comment

**Review by heinezen — CHANGES_REQUESTED** [review]

File: src/data/faq_de.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq_de.json]: Like this @heinezen?

**Comment by Ein-Tim** [comment]
Did it @heinezen. Is this how you wanted it?

**Comment by svengabr** [comment]
With your latest changes it will look like this:

ein-tim

Correct would be to change
`<b>UPDATE</b><br/>:`
to
`<b>UPDATE:</b><br/>`

But to keep consistency, I will remove the colon completely.

consistent

**Pull request merged**